ironblogger/static/style.css

```diff
@@ -81,6 +81,12 @@ section {
   overflow:hidden;
 }
 
+section > p > a {
```

Comment:
So I'm not really a fan of the fact that this is is targeting anchors, since it's meant to affect images. Is there a reason for this? At a glance, `section > p img` and then leaving the width along and setting `margin-{left,right}` to `auto` seems to have the same effect when I test it locally.